scipy/stats/_qmc.py

```diff
@@ -914,13 +910,48 @@ class LatinHypercube(QMCEngine):
 
         .. versionadded:: 1.8.0
 
+    strength : {1, 2}, optional
+        Strength of the LHS. ``strength=1`` produces a plain LHS while
+        ``strength=2`` produces an orthogonal array based LHS of strength 2
+        [7]_, [8]_. In that case, only ``n=p**2`` points can be sampled,
+        with ``p`` a prime number. It also constrains ``d <= p + 1``.
+        Default is 1.
+
+        .. versionadded:: 1.8.0
+
     seed : {None, int, `numpy.random.Generator`}, optional
         If `seed` is None the `numpy.random.Generator` singleton is used.
         If `seed` is an int, a new ``Generator`` instance is used,
         seeded with `seed`.
         If `seed` is already a ``Generator`` instance then that instance is
         used.
 
+    Notes
+    -----
+
+    When LHS is used for integrating a function :math:`f` over :math:`n`,
+    LHS is extremely effective on integrands that are nearly additive [2]_.
+    With a LHS of :math:`n` points, the variance of the integral is always
+    lower than plain MC on :math:`n-1` points [3]_. There is a central limit
+    theorem for LHS on the mean and variance of the integral [4]_, but not
+    necessarily for optimized LHS due to the randomization.
+
+    :math:`A` is called an orthogonal array of strength :math:`t` if in each
+    n-row-by-t-column submatrix of :math:`A`: all :math:`p^t` possible
+    distinct rows occur the same number of times,
+    with :math:`p` a prime number. The elements of :math:`A` are in the set
+    :math:`\{0, 1, ..., p\}`, also called symbols.
+
+    To create an orthogonal array based LHS, the array is randomized by
+    randomly permuting the symbol values on each column independently. Then,
```

Comment:
It sounds to me like this translates to:

for each column of `orthogonal_array`:
   randomly permute column

That doesn't sound like it would preserve the strength.

Reply:
I see the difficulty. I think the example helps. Perhaps I will consider this further in another PR.

Reply:
I tried to add an illustrative example. If we want to do more, then I guess we need to go with a full step by step example. I can do that if you want. Not sure though if we want to be that detailed. I will let you judge on this one.